packages/common/src/authentication/AuthContext.tsx

```diff
@@ -147,7 +148,41 @@ export const AuthProvider: FC<PropsWithChildrenOnly> = ({ children }) => {
     return !!stores && stores?.length > 0 ? stores?.[0] : undefined;
   };
 
+  const setLoginError = (isLoggedIn: boolean, hasValidStore: boolean) => {
+    switch (true) {
+      case isLoggedIn && hasValidStore: {
+        setError(undefined);
+        break;
+      }
+      case !hasValidStore: {
+        setError(AuthError.NoStoreAssigned);
+        break;
+      }
+      default: {
+        setError(AuthError.Unauthenticated);
+      }
+    }
+  };
+
+  const authNameQueries = ['authToken', 'me'];
+  const isAuthRequest = (definitionNode: DefinitionNode) => {
```

Comment:
this bit is app specific and shouldn't be in common.. but that's where the auth context currently is. I've lifted out of GqlContext.. ideally this would all be in some app-specific common package. that will come soon..